emails/daily_email_text.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each items}}
+
+## {{ title }}
+{{#each entries}}
+- {{ title }}<{{ url }}>
```

Comment:
```suggestion
- {{ title }} - {{ url }}
```
<img width="848" alt="Screenshot 2023-11-26 at 18 03 09" src="https://github.com/eduardostuart/paperboy/assets/767879/fc1cbf95-5d57-4644-9f7d-06044bcfe0ea">

Could you change this, please? With the `<` & `>` it is really hard to read.

Reply:
yes, I tried this format because of https://stackoverflow.com/a/61786168/5155484